packages/@adobe/react-spectrum/package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "prepublishOnly": "mkdir -p dist; cp src/index.ts dist/types.d.ts; grep -v '^export type' src/index.ts > dist/module.js; babel --root-mode upward src/index.ts -o dist/main.js"
+    "prepublishOnly": "mkdir -p dist; cp src/index.ts dist/types.d.ts; sed -i '/use client;/d' dist/types.d.ts; grep -v '^export type' src/index.ts > dist/module.js; babel --root-mode upward src/index.ts -o dist/main.js"
```

Comment:
I'm not sure this is clearing it?
https://sed.js.org/

Reply:
Hmmm may need to make sure we're escaping the single quotes correctly

Reply:
Looks like this should get it: `'s/'\''use client'\'';//'`